recipe/meta.yaml

```diff
@@ -24,10 +24,10 @@ requirements:
     - {{ pin_compatible('distributed', max_pin='x.x.x') }}
     - cytoolz >=0.8.2
     - lz4
-    - numpy >=1.18
-    - pandas >=1.0
+    - numpy >=1.21
+    - pandas >=1.3
     - bokeh >=2.4.2,<3
-    - jinja2
+    - jinja2 >=2.10.3
```

Comment:
Seems like this was a change made with https://github.com/dask/distributed/pull/7285 that we missed in the 2023.1.0 release? cc @jrbourbeau @jakirkham just making sure that leaving this unconstrained wasn't intentional in the past few releases.

Shouldn't be difficult to submit a repodata patch to update this for the impacted versions

Reply:
Good catch! Should we update [the `jinja` optional requirement in `dask`]( https://github.com/dask/dask/blob/46c47be15661dd885e32d81f34d6d2bf114d52c8/setup.py#L21 ) as well?

Reply:
Yeah good catch, bumping that makes sense to me

Reply:
Opened up https://github.com/dask/dask/pull/9999 to handle this